broken-line/contracts/LibBrokenLine.sol

```diff
@@ -51,8 +65,51 @@ library LibBrokenLine {
     }
 
     /**
-     * Обновляет initial линию для переданного time. Высчитывает и применяет все изменения из slopeChanges за этот период
-    **/
+     *remove Line from BrokenLine, return line.bias, which actual now moment
+     **/
+    function remove(BrokenLineDomain.BrokenLine storage brokenLine, uint id, uint toTime) internal returns (uint) {
+        BrokenLineDomain.LineData memory lineData = brokenLine.initiatedLines[id];
+        require(lineData.line.bias != 0, "Line with given id already finished");
```

Comment:
why line.bias is checked? in add function line.start is checked (not consistent)

Reply:
agree,  remove require
add function line.start is checked to prevent recurrence adding line with the same id